dynawo/3rdParty/openmodelica/patch/common/openmodelica.patch

```diff
@@ -2157,11 +2157,11 @@ index 000000000..d9c2e1570
 +  end matchcontinue;
 +end createDAE2Dynawo;
 +
-+//================================================================================
-+//   A.D :  Programmation en cours pour regroupement des struct => A debbuger
-+//================================================================================
++//=====================================================================================
```

Comment:
Directly editing a patch is generally not a good idea

Reply:
These are comments, what could go wrong ?

Reply:
windows is very sensitive to the index given after the file name ` index f6bc18ab2e..6eccb4f6ec 100644`

Reply:
Agreed, I have restored the original version.